application/squeeze2cast/squeeze2cast.c

```diff
@@ -917,6 +942,9 @@ static bool AddCastDevice(struct sMR *Device, char *Name, char *UDN, bool group,
 	Device->NextMime[0]	 	= '\0';
 	Device->NextURI 		= NULL;
 	Device->Group 			= group;
+	Device->Expired         = 0;
```

Comment:
Wasn't that the real mistake I made? Not initializing "Expired"

Reply:
That was the first change I made (I had forgotten). But even if necessary, it didn't immediately fix all the problems.

Reply:
So I'd have to rethink the if() logic then